### PR TITLE
i5879: Added a standalone example

### PR DESCRIPTION
**Pull request by prasun3**
i5879: Added a standalone example for creating new analysis tools

Issue: #5879

### CODE REVIEW

**Comment by derekbruening** [comment]
add to allowlist

**Comment by derekbruening** [comment]
run arm tests

**Comment by derekbruening** [comment]
@joshua-warburton is it possible the PR Jenkins trigger is no longer working after the merge queue changes?  It didn't seem to trigger here and @prasun3 was already on the allowlist.

File: clients/drcachesim/standalone_analyzer_example/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.10)
```
Comment (derekbruening) [inline clients/drcachesim/standalone_analyzer_example/CMakeLists.txt:1]: > i5879: Added a standalone example

Please specify what kind of example in the title to make it more descriptive: include "drmemtrace analyzer" or something.

File: clients/drcachesim/standalone_analyzer_example/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+project(analyzer_example)
```
[the remark this answers is not in the document]
Comment (derekbruening) [reply clients/drcachesim/standalone_analyzer_example/CMakeLists.txt:2]: Another question is whether the example should be exported in the release package.

**Comment by joshua-warburton** [comment]
@derekbruening, I didn't make any changes to the existing pipeline, so it shouldn't have changed.

**Comment by joshua-warburton** [comment]
Looks like somethings gone wrong with the authentication with github:
```ould not update commit status of the Pull Request on GitHub.
org.kohsuke.github.HttpException: {"message":"`DynamoRIO` forbids access via a personal access token (classic). Please use a GitHub App, OAuth App, or a personal access token with fine-grained permissions.","documentation_url":"https://docs.github.com/rest/commits/statuses#create-a-commit-status"}
	at org.kohsuke.github.GitHubConnectorResponseErrorHandler$1.onError(GitHubConnectorResponseErrorHandler.java:56)
	at org.kohsuke.github.GitHubClient.detectKnownErrors(GitHubClient.java:424)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:386)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:355)
	at org.kohsuke.github.Requester.fetch(Requester.java:76)
	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:2108)
	at org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus.createCommitStatus(GhprbSimpleStatus.java:283)
	at org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus.onBuildComplete(GhprbSimpleStatus.java:241)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:205)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:28)
	at hudson.model.listeners.RunListener.lambda$fireCompleted$0(RunListener.java:207)
	at jenkins.util.Listeners.lambda$notify$0(Listeners.java:59)
	at jenkins.util.Listeners.notify(Listeners.java:67)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:205)
	at hudson.model.Run.execute(Run.java:1941)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
	at hudson.model.ResourceController.execute(ResourceController.java:101)
	at hudson.model.Executor.run(Executor.java:442)
```

**Comment by prasun3** [comment]
Oh I saw some recent change about fine-grained permission. I haven't set it up yet.

**Comment by derekbruening** [comment]
We did disable non-fine-grained personal access tokens (did not do a broader announcement as we thought they were only being used in the docs automation)

**Pull request closed without merging**